ydb/docs/en/core/reference/ydb-sdk/health-check-api.md

```diff
@@ -7,9 +7,31 @@ description: "The article will tell you how to initiate the check using the Heal
 
 {{ ydb-short-name }} has a built-in self-diagnostic system, which can be used to get a brief report on the database status and information about existing problems.
 
-To initiate the check, call the `SelfCheck` method from `Ydb.Monitoring`. You must also pass the name of the checked DB as usual.
+To initiate the check, call the `SelfCheck` method from SDK `Ydb.Monitoring`. You must also pass the name of the checked DB as usual.
```

Comment:
it sounds like `Ydb.Monitoring` is a class name from some other programming language, while example below uses `TMonitoringClient`

```suggestion
To initiate the check, call the `SelfCheck` method from a monitoring client bundled with {{ ydb-short-name }} SDK. You must also pass the name of the checked DB as usual. Examples:
```